python/tk_multi_loader/delegate_publish_history.py

```diff
@@ -17,6 +17,107 @@
 shotgun_model = sgtk.platform.import_framework("tk-framework-shotgunutils", "shotgun_model")
 shotgun_view = sgtk.platform.import_framework("tk-framework-qtwidgets", "shotgun_view")
 
+from .ui.widget_publish_history import Ui_PublishHistoryWidget
+
+class PublishHistoryWidget(QtGui.QWidget):
+    """
+    Simple list item widget which hosts a square thumbnail, header text
+    and body text. It has a fixed size. Used with SgPublishHistoryDelegate
+    and used in the right hand side history UI. 
+    """
+    
+    def __init__(self, parent):
+        """
+        Constructor
+        
+        :param parent: QT parent object
+        """
+        QtGui.QWidget.__init__(self, parent)
+
+        # make sure this widget isn't shown
+        self.setVisible(False)
```

Comment:
I suspect I know why you are having to do this (I think I've just run into the same thing) - we should probably fix this in the base delegate class ;)